vnext/Microsoft.ReactNative/Views/TextViewManager.cpp

```diff
@@ -128,6 +130,34 @@ class TextShadowNode final : public ShadowNodeBase {
     this->GetView().as<xaml::Controls::TextBlock>().TextHighlighters().Append(newHigh);
   }
 
+  void ToggleTouchEvents(XamlView xamlView, bool selectable) {
+    if (selectable) {
+      if (m_touchEventHandler == nullptr) {
+        m_touchEventHandler = std::make_unique<TouchEventHandler>(GetViewManager()->GetReactContext());
+      }
+
+      auto selectionChanged = std::make_shared<bool>(false);
+      std::function<bool()> shouldCancelOnCaptureLost = [selectionChanged]() {
+        const auto wasSelectionChanged = *selectionChanged;
+        *selectionChanged = false;
+        return wasSelectionChanged;
+      };
+
+      m_selectionChangedRevoker = xamlView.as<xaml::Controls::TextBlock>().SelectionChanged(
```

Comment:
We should use the SelectionChanging event, as this will occur synchronously. There's likely a small possibility that we could highlight exactly one character and release the pointer and the pointer event could fire before SelectionChanged (which fires asynchronously from the actual selection change after the text highlight renders).

Reply:
Actually, TextBlock doesn't have a synchronous SelectionChanging event, so we're out of luck here.